service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -331,6 +342,18 @@ private String doPasswdAuth(HttpServletRequest request, String authType)
     return userName;
   }
 
+  private String doTokenAuth(HttpServletRequest request, HttpServletResponse response)
+      throws HttpAuthenticationException {
+
+    String delegationTokenHeader = "X-Hadoop-Delegation-Token";
```

Comment:
Should we use DelegationTokenAuthenticator.DELEGATION_TOKEN_HEADER here like in the constructor above?